### PR TITLE
Mycnf edits

### PR DESCRIPTION
**Pull request by rnirmal**
Added back local-infile
Added a max_user_connections and max_connections = max_user_connections + 10

### CODE REVIEW

**Comment by hub-cap** [comment]
Saweet.

File: contrib/dbaas-mycnf/builddeb.sh [hunk 1]
```diff
@@ -35,7 +35,12 @@ for i in "${MEMSIZE[@]}"; do
     cat ../etc/my.cnf.base | while read line; do
         if [[ `expr "$line" : ".*{.*}"` != "0" ]]; then
             oldval=`echo $line | sed -e 's/.*{\(.*\)}.*/\1/'`
-            newval=`echo "$oldval * $multiplier" | bc`
+            prop=`echo $line | sed -e 's/^\(.*\) = {100}/\1/'`
```
Comment (ed-) [inline contrib/dbaas-mycnf/builddeb.sh:38]: I wondered for a moment why you were shelling out to echo | bc to do something like multiply numbers until I looked again and saw this is a shell script

Details matter!

**Comment by ed-** [comment]
+1

File: contrib/dbaas-mycnf/builddeb.sh (hunk 1 above)
Comment (rnirmal) [inline contrib/dbaas-mycnf/builddeb.sh:38]: :)

**Comment by hub-cap** [comment]
+1 pending squash

**Comment by TimSimpsonR** [comment]
+1

**Comment by cp16net** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**